index.html

```diff
@@ -1113,6 +1113,10 @@
 <p><span class="practicelab" id="principle-safety-with-privacy">As much as possible,
 services should provide both safety and privacy.</span></p>
 
+This may require the development and adoption of new technologies. For example, wider
+adoption of [[?WebAuthn]] could reduce account hijacking enough that services no longer
+need to fingerprint devices to prevent it.
+
```

Comment:
Consider mentioning: 
- However, not all safety problems can be solved through new technologies. Certain systems that combine strong privacy with public safety risk may be impossible to make safe.
Examples I have in mind are WhatsApp with large fan-out, and systems like 8chan that are built around a strategy to prevent accountability.

Reply:
This text is no longer in the PR.